packages/extension_google_sign_in_as_googleapis_auth/example/android/gradle.properties

```diff
@@ -1,3 +1,3 @@
-org.gradle.jvmargs=-Xmx1536M
```

Comment:
I recall there are actually 2 parameters you need to set here. Let me see if I can find an example.

Reply:
I found the commit I think: https://github.com/flutter/packages/commit/0814a3b5374767cc0fd1f0325f0290361dff22f6

Reply:
Ok, this is the same issue as https://github.com/flutter/flutter/issues/154489. It seems like upgrading the guava version in `6.1.25` of `google_sign_in_android` has caused some issues with our use of R8 (I verified that reverting back to using 
```
implementation 'com.google.guava:guava:32.0.1-android'
```
makes this example app build successfully.

I think perhaps we should revert back to that guava version to unblock the roller (and also to unblock `google_sign_in_android` being broken on flutter's latest master, and then investigate what it will take to unblock that upgrade

Reply:
Done, and updated branch

Reply:
Looks like it builds successfully now!